packages/trader/src/AppV2/Components/Guide/guide.tsx

```diff
@@ -20,7 +19,7 @@ const Guide = observer(({ has_label, show_guide_for_selected_contract }: TGuide)
         ui: { is_dark_mode_on },
     } = useStore();
     const { contract_type, is_vanilla } = useTraderStore();
-    const contract_type_title = is_vanilla ? CONTRACT_LIST.VANILLAS : getContractTypesConfig()[contract_type]?.title;
+    const contract_type_title = is_vanilla ? CONTRACT_LIST.VANILLAS : getContractTypesList()[contract_type];
```

Comment:
Why do we have a check for `is_vanilla` specifically? Shouldn't we just get it from `getContractTypesList()[contract_type]`?

Reply:
Agree, sorry, my bad. 
`getContractTypesConfig()[contract_type]?.title` was not returning 'Vanillas', it returns 'Call/Put'. After I replace old function with new one (`getContractTypesList()[contract_type]`), which I created, this problem is solved. In new function we return 'Vanillas'